server/status/recorder_test.go

```diff
@@ -276,8 +279,14 @@ func TestNodeStatusRecorder(t *testing.T) {
 		generateStoreData(2, "capacity.available", 100, 75),
 
 		// Node stats.
-		generateNodeData(1, "calls.success", 100, 2),
-		generateNodeData(1, "calls.error", 100, 1),
+		generateNodeData(1, "exec.successcount", 100, 2),
```

Comment:
These names need a delimiter, and I still think the metric package should add one. It doesn't need to be the *same* delimiter if that would alleviate your concern about codifying our naming conventions: `exec.success@count`, `exec.success@1h`.